src/us/kbase/workspace/database/ByteArrayFileCacheManager.java

```diff
@@ -338,7 +237,7 @@ public void destroy() {
 			} catch (IOException ioe) {
 				//nothing can be done
 			}
-			if (tempFile != null && tempFile.exists()) {
+			if (tempFile != null) {
 				tempFile.delete();
```

Comment:
If the tempfile doesn't exist `delete()` is a noop, so no need to check for existence